code/modules/antagonists/heretic/status_effects/debuffs.dm

```diff
@@ -192,3 +192,25 @@
 
 /datum/status_effect/star_mark/extended
 	duration = 3 MINUTES
+
+// Last Resort
+/datum/status_effect/heretic_lastresort
+	id = "heretic_lastresort"
+	alert_type = /atom/movable/screen/alert/status_effect/heretic_lastresort
+	duration = 12 SECONDS
+	status_type = STATUS_EFFECT_REPLACE
+	tick_interval = -1
+
+/atom/movable/screen/alert/status_effect/heretic_lastresort
+	name = "Last Resort"
+	desc = "Your head spins, heart pumping as fast as it can, losing the fight with the ground, run to safety!"
+	icon_state = "lastresort"
+
+/datum/status_effect/heretic_lastresort/on_apply()
+	ADD_TRAIT(owner, TRAIT_IGNORESLOWDOWN, MAGIC_TRAIT)
+	to_chat(owner, span_userdanger("You are on the brink of losing consciousness, run!"))
+	return TRUE
+
+/datum/status_effect/heretic_lastresort/on_remove()
+	REMOVE_TRAIT(owner, TRAIT_IGNORESLOWDOWN, MAGIC_TRAIT)
+	owner.AdjustUnconscious(20 SECONDS, ignore_canstun = TRUE)
```

Comment:
```suggestion
	owner.AdjustUnconscious(20 SECONDS, ignore_canstun = TRUE)
```
That's a pretty long time, even 5 or 10 seconds is probably enough that if you failed to snap your blade you're definitely going to get caught again 😅

Reply:
its a downside to get out of jail free